apps/publishers/tests/services/test_publisher_member_invitation_service.py

```
@@ -91,7 +93,7 @@ def test_old_token_is_dead_after_resend(self):
         self.assertEqual("invalid_invitation", ctx.exception.error_name)
 
     def test_accept_admin_activates_sets_password_and_grants_perms(self):
-        member, inv, raw, _ = self._create(email="newbie@example.com", role=PublisherMember.RoleChoice.ADMIN)
+        member, inv, raw, _ = self._create(email="newbie@example.com", group=admin_group())
```

Ruff (0.16.1)

[warning] Unpacked variable `member` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)